db/fill-db.sh

```diff
@@ -5,10 +5,10 @@
 SETUP_DIR=/var/lib/postgresql/setup
 TABLES=schema.sql
 
-echo "$0: running $TABLES"; psql -d "$POSTGRES_DB" -f "$SETUP_DIR/$TABLES";
+echo "$0: running $TABLES"; psql -d "$POSTGRES_DB" -U "$POSTGRES_USER" -f "$SETUP_DIR/$TABLES";
```

Comment:
Use the user from the `POSTGRES_USER` environment variable, by default it was using the `postgres` user.